### PR TITLE
Add initial Pie Chart implementation

### PR DESCRIPTION
**Pull request by JohnBarton27**
I wanted to add a Pie Chart to a Sheet I was populating using pygsheets. I added an initial implementation here that extends the `Chart` class with a few key changes. Definitely a little bit of duplicated/similar code in the methods that get overridden in the `PieChart` class - open to suggestions if you want a deeper re-write to abstract some of that out. Hopefully this can also help build a pattern for other future "non-basic" Charts to get support.

### CODE REVIEW

File: pygsheets/worksheet.py [hunk 1]
```diff
@@ -1676,6 +1677,19 @@ def add_chart(self, domain, ranges, title=None, chart_type=ChartType.COLUMN, anc
         """
         return Chart(self, domain, ranges, chart_type, title, anchor_cell)
 
+    def add_pie_chart(self, domain, chart_range, title=None, anchor_cell=None, three_dimensional=False, pie_hole=0):
+        """
+        Similar to `add_cart`, but created a Pie Chart instead of a Basic Chart.
```
Comment (nithinmurali) [inline pygsheets/worksheet.py]: typo:  add_cart should be add_chart

File: tests/online_test.py [hunk 2]
```diff
@@ -1234,6 +1292,11 @@ def test_start_ub_end(self):
         assert self.grange.label == "'" + self.worksheet.title + "'" + '!' + '1' + ':' + '4'
 
 
+class TestPieChart(object):
```
Comment (nithinmurali) [inline tests/online_test.py]: If there are no methods in this class, remove this?

**Review by nithinmurali — APPROVED** [review]
Overall looks good. just some minor comments.

**Comment by JohnBarton27** [comment]
> Overall looks good. just some minor comments.

Sounds good, just pushed fixes!

**Comment by nithinmurali** [comment]
Awesome! thanks for the PR.

**Pull request merged**